src/Silex/Application.php

```diff
@@ -299,7 +298,7 @@ public function finish($callback, $priority = 0)
     {
         $app = $this;
 
-        $this->on(KernelEvents::TERMINATE, function (PostResponseEvent $event) use ($callback, $app) {
+        $this->on(KernelEvents::TERMINATE, function (ResponseEvent $event) use ($callback, $app) {
```

Comment:
https://github.com/symfony/http-kernel/blob/v4.4.50/Event/PostResponseEvent.php

```
/**
 * @deprecated since Symfony 4.3, use TerminateEvent instead
 */
class PostResponseEvent extends KernelEvent
```